widget/list.go

```diff
@@ -93,6 +93,22 @@ func (l *List) MinSize() fyne.Size {
 	return l.BaseWidget.MinSize()
 }
 
+func (l *List) RefreshItem(id ListItemID) {
```

Comment:
Please add a comment for new plunking APIs. Look at the function below this one to see how it is supposed to be done (substituting the version with 2.4 in this case).